cis.py

```diff
@@ -48,15 +45,17 @@ def get_parser():
                              'subfolder in dset_dir.')
     parser.add_argument('--config', required=True, dest='config',
                         help='Path to the config json file.')
-    parser.add_argument('--protocol_check', required=False, action='store_true',
-                        help='Will perform a protocol check to determine \
-                             if the correct number of scans and TRs are present.')
+    parser.add_argument('--protocol_check', required=False,
+                        action='store_true',
+                        help='Will perform a protocol check to determine if \
```

Comment:
with multi-line strings, do the following instead of using backslashes:

```python
help=('Something here. '
      'And we continue the line.')
```

Reply:
ok done